src/special_arrays.jl

```diff
@@ -82,30 +87,42 @@ end
 function transform_with(flag::LogJacFlag, t::UnitVector, x::AbstractVector, index)
     (; n) = t
     T = robust_eltype(x)
-    log_r = zero(T)
     y = Vector{T}(undef, n)
-    ℓ = logjac_zero(flag, T)
-    @inbounds for i in 1:(n - 1)
-        xi = x[index]
-        index += 1
-        y[i], log_r, ℓi = l2_remainder_transform(flag, xi, log_r)
-        ℓ += ℓi
-    end
-    y[end] = exp(log_r / 2)
+    z = view(x, index:index+n-1)
+    r = norm(z)
+    copyto!(y, z)
```

Comment:
This could be shortened to
```suggestion
    copyto!(y, 1, x, index, n)
    r = norm(y)
```

Reply:
Thanks! I always forget these methods exist.